README.md

```diff
@@ -109,10 +138,3 @@ In the popup-select the "NativeScript" option and set the class name to "HelloWo
 **NOTE**: After creation, the NativeScript resource does not automatically point to the `GDNativeLibrary` resource.
 Make sure to set click the "library" field in the Inspector and "load" the library.
 
-## Contributing
-
-See the [contribution guidelines](CONTRIBUTING.md)
-
-## License
-
-Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in the work by you shall be licensed under the [MIT license](LICENSE.md), without any additional terms or conditions.
```

Comment:
Personally, I tend to scroll straight to the bottom when I'm looking for license info, since it's usually there below everything.

I think it's better to put it either at the top or at the bottom: not many projects put their license info in the middle.